cli/cmd/service/create.go

```diff
@@ -17,10 +23,43 @@ func CreateCmd(ch *cmdutil.Helper) *cobra.Command {
 				return err
 			}
 
-			res1, err := client.CreateService(cmd.Context(), &adminv1.CreateServiceRequest{
+			// Parse attributes if provided
+			var attrs map[string]string
+			if attributes != "" {
+				if err := json.Unmarshal([]byte(attributes), &attrs); err != nil {
+					return fmt.Errorf("failed to parse --attributes as JSON: %w", err)
+				}
+			}
+
+			req := &adminv1.CreateServiceRequest{
 				Name:             args[0],
 				OrganizationName: ch.Org,
-			})
+			}
+
+			if orgRole == "" && projectRole == "" {
+				return fmt.Errorf("either --org-role or --project-role must be specified")
+			}
```

Comment:
It's not ideal to require a flag (since flags are usually optional). What do you think would be a nice behavior here?

Reply:
Another option is to to prompt for whether they want to add an org role or not and give role options to select from, similar for project name and roles. That will solve [this](https://github.com/rilldata/rill/pull/7480#discussion_r2178048137) as well.

Reply:
changed the flow please check